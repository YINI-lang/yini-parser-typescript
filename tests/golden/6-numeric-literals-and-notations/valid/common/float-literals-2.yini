^ Floats
float1 = .5       // leading decimal point without a zero
float2 = -.25     // negative with leading decimal point
float3 = 0.0      // zero as float
float4 = 5.00
float5 = -0.0     // negative zero as float
