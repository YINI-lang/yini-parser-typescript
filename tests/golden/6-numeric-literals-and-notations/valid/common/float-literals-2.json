{
    "Floats": {
        "float1": 0.5,
        "float2": -0.25,
        "float3": 0.0,
        "float4": 5.0,
        "float5": 0.0
    }
}
